static/app/views/releases/drawer/generalCard.tsx

```diff
@@ -0,0 +1,132 @@
+import styled from '@emotion/styled';
+
+import Count from 'sentry/components/count';
+import {DateTime} from 'sentry/components/dateTime';
+import {Card} from 'sentry/components/keyValueData';
+import Link from 'sentry/components/links/link';
+import LoadingError from 'sentry/components/loadingError';
+import Placeholder from 'sentry/components/placeholder';
+import TimeSince from 'sentry/components/timeSince';
+import {t, tn} from 'sentry/locale';
+import type {ReleaseMeta} from 'sentry/types/release';
+import useOrganization from 'sentry/utils/useOrganization';
+import {isVersionInfoSemver} from 'sentry/views/releases/utils';
+import {useReleaseDetails} from 'sentry/views/releases/utils/useReleaseDetails';
+
+interface DetailsSectionProps {
+  isMetaError: boolean;
+  projectSlug: string;
+  release: string;
+  releaseMeta?: ReleaseMeta;
+}
+
+export function GeneralCard({
+  isMetaError,
+  projectSlug,
+  release,
+  releaseMeta,
+}: DetailsSectionProps) {
+  const {isError: isDetailsError, data: releaseDetails} = useReleaseDetails({
+    release,
+  });
+  const organization = useOrganization();
+
+  if (isDetailsError) {
+    return <LoadingError />;
+  }
+
+  const generalContentItems = [
+    {
+      item: {
+        key: t('Date Created'),
+        subject: t('Date Created'),
+        value: releaseDetails ? (
+          <DateTime date={releaseDetails.dateCreated} />
+        ) : (
+          <TinyPlaceholder />
+        ),
+      },
+    },
+    {
+      item: {
+        key: t('Semver'),
+        subject: t('Semver'),
+        value: releaseDetails ? (
+          isVersionInfoSemver(releaseDetails.versionInfo.version) ? (
+            t('Yes')
+          ) : (
+            t('No')
+          )
+        ) : (
+          <TinyPlaceholder />
+        ),
+      },
+    },
+    {
+      item: {
+        key: t('Package'),
+        subject: t('Package'),
+        value: releaseDetails ? (
+          (releaseDetails.versionInfo.package ?? '\u2014')
+        ) : (
+          <TinyPlaceholder />
+        ),
+      },
+    },
+    {
+      item: {
+        key: t('First Event'),
+        subject: t('First Event'),
+        value: releaseDetails ? (
+          <TimeSince date={releaseDetails.firstEvent} />
+        ) : (
+          <TinyPlaceholder />
+        ),
+      },
+    },
+    {
+      item: {
+        key: t('Last Event'),
+        subject: t('Last Event'),
+        value: releaseDetails ? (
+          <TimeSince date={releaseDetails.lastEvent} />
+        ) : (
+          <TinyPlaceholder />
+        ),
+      },
+    },
+    {
+      item: {
+        key: t('Source Maps'),
+        subject: t('Source Maps'),
```

Comment:
We'll want this to be dynamic, mobile apps don't have source maps but they do have other artifact types.